recipes/tvb-data/meta.yaml

```diff
@@ -0,0 +1,39 @@
+
+package:
+  name: tvb-data
+  version: "1.5.4"
+
+source:
+  git_rev: 805f9af4eb05722a52db789307bb81e261ba529a
```

Comment:
Please [build from tarballs, not repos](http://conda-forge.readthedocs.io/en/latest/meta.html#build-from-tarballs-not-repos).